src/common/lang.js

```diff
@@ -135,6 +135,7 @@ const lang = {
   INFO: 'Info',
   JOBNAME: 'Job Name',
   JOB_NAME: 'Job Name',
+  JOB_NAME_REFERENCE: 'Job Name will not display spaces and special characters',
```

Comment:
I think this string makes more sense in the context of a form. 

`JOB_NAME_REFERENCE: 'Only letters and numbers are allowed in the job name',`

Reply:
These are the comps provided by Caitlin.
![image](https://user-images.githubusercontent.com/16091688/32189658-a66d9bc8-bd68-11e7-8148-77c9169d2d2c.png)